cavern/src/main/java/org/opencadc/cavern/files/CavernURLGenerator.java

```diff
@@ -356,6 +348,16 @@ public VOSURI validateToken(String token, VOSURI targetVOSURI, Direction directi
         throw new IllegalArgumentException("Missing node URI");
     }
 
+    URL getFilesEndpoint() {
+        String hack = "https://haproxy.cadc.dao.nrc.ca/cavern/files";
+        try {
```

Comment:
Is this hack here on purpose?

Reply:
just temporary to get it working... now I am at the point where I can get it dynamically. will fix